process/check.py

```diff
@@ -342,9 +342,13 @@ def check(self, instance):
             # psutil.NoSuchProcess is raised.
             pids = self._get_pid_set(pid)
         elif pid_file is not None:
-            with open(pid_file, 'r') as file_pid:
-                pid_line = file_pid.readline().strip()
-                pids = self._get_pid_set(int(pid_line))
+            try:
+                with open(pid_file, 'r') as file_pid:
+                    pid_line = file_pid.readline().strip()
+                    pids = self._get_pid_set(int(pid_line))
+            except IOError:
```

Comment:
Maybe add a info log. It will make support much easier when a client enter the wrong path.